content/ja/docs/contribute/new-content/overview.md

```diff
@@ -0,0 +1,54 @@
+---
+title: 新しいコンテンツの貢献の概要
+linktitle: 概要
+content_type: concept
+main_menu: true
+weight: 5
+---
+
+<!-- overview -->
+
+このセクションでは、新しいコンテンツの貢献を行う前に知っておくべき情報を説明します。
+
+<!-- body -->
+
+## 貢献の基本
+
+- KubernetesのドキュメントはMarkdownで書き、Kubernetesのウェブサイトは[Hugo](https://gohugo.io/)を使ってビルドします。
+- ソースは[GitHub](https://github.com/kubernetes/website)にあります。Kubernetesのドキュメントは`/content/en/docs/`にあります。リファレンスドキュメントの一部は、`update-imported-docs/`ディレクトリ内のスクリプトから自動的に生成されます。
+- [Page content types](/docs/contribute/style/page-content-types/)はHugo内のドキュメントのコンテンツの見え方を？指定します？
```

Comment:
It is a difficult part for translation.. :sweat_smile: 
How about my suggest following?
```suggestion
- [Page content types](/docs/contribute/style/page-content-types/)にHugoによるドキュメントのコンテンツの見え方を記述しています。
```

Reply:
I'm sorry, I thought I would fix it later but forgot it... 😅

I think your translation is perfect.